rfc/legacy_artifacts_ui_removal.md

```diff
@@ -0,0 +1,131 @@
+# Legacy Artifacts UI Removal
+
+| | |
+|-|-|
+| **Status**     | _**Proposed**, Accepted, Implemented, Obsolete_ |
+| **RFC #**      | [111](https://github.com/spinnaker/governance/pull/111) |
+| **Author(s)**  | Maggie Neterval (`@maggieneterval`) |
+| **SIG / WG**   | UI/UX SIG |
+| **Obsoletes**  | [spinnaker/spinnaker/4777](https://github.com/spinnaker/spinnaker/issues/4777) |
+
+## Overview
+
+Spinnaker currently supports two artifact configuration experiences: the legacy
+UI and a feature-flagged "rewrite" UI. This RFC outlines a plan to test the new
+UI, make it the default artifact configuration experience, and remove the legacy
+UI.
+
+### Vocabulary
+
+The rest of this RFC will refer to the two UIs as the _legacy UI_ and the _new
+UI_:
+
+- Legacy UI: Enabled by setting Deck's `artifacts` feature flag to `true`.
+Disabled by default. The
+[documentation](https://www.spinnaker.io/reference/artifacts/) currently
+refers to this UI as the "standard UI", but it has also been referred to as the
+"pre-rewrite UI."
+
+- New UI: Enabled by setting Deck's `artifactsRewrite` feature flag to `true`.
+Disabled by default. The
```

Comment:
Sorry for the annoying question, but if both the legacy and new UIs are controlled by feature flags (and disabled by default), what do you get when both are disabled?

Reply:
Not an annoying question at all, David! If both feature flags are disabled (which is currently the default), there is no Expected Artifacts section when configuring a pipeline, and the Find Artifact from Resource stage and Find Artifact from Execution stage are not available.